src/CMakeLists.txt

```diff
@@ -174,6 +178,6 @@ install(TARGETS umf EXPORT ${PROJECT_NAME}-targets)
 
 add_subdirectory(pool)
 
-if(UMF_PROXY_LIB_ENABLED)
+if(UMF_PROXY_LIB_ENABLED AND NOT UMF_LINK_HWLOC_STATICALLY)
```

Comment:
soo.. now if hwloc is linked statically the proxy_lib won't build...? we should add a message with such info (e.g. around the place where `UMF_PROXY_LIB_ENABLED` is set; or just here)

Reply:
when both proxy lib and UMF are linked statically with hwloc, we have a problem with doubling the hwloc symbols. But in the scenario that we want to enable the proxy lib wouldn't be used

Reply:
status message shouldn't hurt, I guess

also, we can write what you just wrote as a comment in this line

Reply:
I've added earlier